### ISSUE TITLE
inline `<script>` marker detection not correctly matching when Rollup bundles the inline source and changes its contents

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#586]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
related to #572, noticed that this is till not exactly dialed in.  Noticed that an existing case was working by accident.

This user code that we would feed to Rollup
```js
// @preserve 756510913-scratch
import '@evergreen-wc/eve-container';
```

Was actually getting bundled by Rollup as this and thus dropping our marker
```js
import"/lit-element.5d109b2b.js";import"/eve-container.2e10f408.js";
```

Because of this, the marker detection logic was only working by accident.  😞 

## Details
Basically our logic here be such that it should _only_ work when `marker === id` which means we need to figure out an alternative marking method.
